package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "type": "module",
```

Comment:
- [ ] While trying to run your project I'm encountering this error: 

<img width="793" alt="Screenshot 2023-06-10 at 9 58 02 AM" src="https://github.com/kerlos-bahaa/to-do-list/assets/44798044/9570e5fd-7f27-4557-828d-614896f7b1ec">

<hr>
Please fix this by removing  `"type": "module"` that is `line:6` from your `package.json` file and replace it with `"private": "true",`. 

- [ ] [OPTIONAL] Also I would highly recommend you to configure your project for opening in server by adding ` "start": "webpack serve --open",` in your `package.json`  scripts. This way you can directly open and view your project live on your server :  Example snippet:

```json

"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "webpack serve --open",
    "build": "webpack"
  },
```